habitat-lab/habitat/config/habitat/task/rearrange/ovmm.yaml

```diff
@@ -0,0 +1,98 @@
+# @package habitat.task
+
+defaults:
+  - /habitat/task: task_config_base
+  - /habitat/task/actions:
+    - rearrange_stop
+    - base_velocity
+    - arm_action
+  - /habitat/task/measurements:
+    - object_to_goal_distance
+    - robot_force
+    - force_terminate
+    - robot_colls
+    - does_want_terminate
+    - num_steps
+    - obj_anywhere_on_goal
+    - end_effector_to_rest_distance
+    - did_pick_object
+    - pick_success
+    - place_success
+    - ovmm_dist_to_pick_goal
+    - ovmm_rot_dist_to_pick_goal
+    - ovmm_nav_to_pick_succ
+    - ovmm_nav_orient_to_pick_succ
+    - ovmm_dist_to_place_goal
+    - ovmm_rot_dist_to_place_goal
+    - ovmm_nav_to_place_succ
+    - ovmm_nav_orient_to_place_succ
+  - /habitat/task/lab_sensors:
+    - joint_sensor
+    - joint_velocity_sensor
+    - object_category_sensor
+    - start_receptacle_sensor
+    - goal_receptacle_sensor
+    - object_segmentation_sensor
+    - object_embedding_sensor
+    - start_recep_segmentation_sensor
+    - goal_recep_segmentation_sensor
+    - robot_start_gps_sensor
+    - robot_start_compass_sensor
+    - camera_pose_sensor
+    - end_effector_sensor
+    - relative_resting_pos_sensor
+    - is_holding_sensor
+    - ovmm_nav_goal_segmentation_sensor
+    - receptacle_segmentation_sensor
+  - _self_
+
+type: CatNavToObjTask-v0
+reward_measure: place_success
+success_measure: place_success
+success_reward: 10.0
+slack_reward: -0.005
+end_on_success: True
+constraint_violation_ends_episode: False
+constraint_violation_drops_object: True
+object_in_hand_sample_prob: 0.0
+actions:
+  arm_action:
+    type: "ArmAction"
+    arm_controller: "ArmRelPosReducedActionStretch"
+    grip_controller: "GazeGraspAction"
+    arm_joint_mask: [1,0,0,0,1,1,1,1,1,1]
+    arm_joint_dimensionality: 7
+    grasp_thresh_dist: 0.15
+    disable_grip: False
+    delta_pos_limit: 6.28 # The range of joints are:  [ 0., 0.,  0.,  0.,  0., -1.75, -1.57, -3.14, -3.9, -1.53] -> [0.13, 0.13, 0.13, 0.13, 1.1 , 4., 0.56, 3.14, 1.5 , 0.79], so maximum delta is 6.28
```

Comment:
I got these values using `cur_robot.arm_joint_limits` in sim. @cpaxton, Jimmy asked me to confirm if these ranges match  real stretch. eg. if the real agent can realize joint_wrist_roll in the entire range [-3.14, 3.14]

Reply:
I think this is correct.